VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/unread-messages/unread-messages.component.ts

```diff
@@ -31,6 +31,14 @@ export class UnreadMessagesComponent implements OnInit, OnDestroy {
                 this.resetConferenceUnreadCounter(message);
             })
         );
+        this.messagesSubscription$.add(
+            this.eventsService.getChatMessage().subscribe(message => {
+                if (message.conferenceId === this.conferenceId) {
```

Comment:
This will not work. The message can be from any one, VHO1/VHO2 or judge. You need to verify the message is from judge

Reply:
It's a good point, that should be taken in account. Currently it does work, we have IMs currently only from judge or VHO/VHO1.... Once we get IM from admin the IMs counter reset to 0, if we get from judge it increase the counter. But it just quick fix as we need to identify users, we really need to know who IMs we have. It could be  more complicated if we will implement IMs for participants as well. I think it would not be enough to know is it Judge or not. We need to create separate task and make relevant changes.